### PR TITLE
Add Uniswap v1 protocol introduction

### PR DESCRIPTION
**Pull request by jackleeio**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] A relating issue is created: #685  (<- fill the issue number here)
- [x] **This is NOT a PR to submit translation**

### CODE REVIEW

File: docs/general/protocols/defi/_category_.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+label: DEFI
```
Comment (0xn1c0) [inline docs/general/protocols/defi/_category_.yml:1]: I'd suggest changing this for "DeFi" instead

**Review by 0xn1c0 — APPROVED** [review]

**Comment by 0xn1c0** [comment]
@darwintree @jackleeio I'm not sure if this content is actually useful or relevant. Which is the purpuse of this "Protocols Introduction Section"? Who is it aimed to? I'd like to know if you have any comments on why you think this content should be here.
Probably a tutorial on how to deploy your own uniswap v1 protocol on conflux would be a lot more relevant for this webpage.

**Comment by jackleeio** [comment]
> Which is the purpuse of this "Protocols Introduction Section"? Who is it aimed to? I'd like to know if you have any comments on why you think this content should be here.


This content was added based on the previous suggestion:https://github.com/Conflux-Chain/conflux-documentation/pull/664#issuecomment-2232253111, and the basic introduction of these protocols is mainly aimed at general users and beginner blockchain developers in the Conflux ecosystem.

> Probably a tutorial on how to deploy your own uniswap v1 protocol on conflux would be a lot more relevant for this webpage.

As for this part, I think it would be good to create an additional tutorial and place it in the tutorial folder

**Comment by 0xn1c0** [comment]
Ok ok, it makes sense, thanks for the clarification!

**Review by 0xn1c0 — APPROVED** [review]

**Review by 0xn1c0 — APPROVED** [review]

**Pull request merged**